test/runner.py

```diff
@@ -389,6 +390,8 @@ def parse_args(args):
                            'Useful when combined with --failfast')
   parser.add_argument('--force64', action='store_const', const=True, default=None)
   parser.add_argument('--crossplatform-only', action='store_true')
+  parser.add_argument('--repeat', type=int, default=1,
+                      help='Repeat each test N times (default: 1).')
```

Comment:
Won't `--help` add the default automatically (i.e. you should n't need to write it out in the help I think).

Reply:
I don't see a default with `--help`